### PR TITLE
BUG: df constructor not copying ea backed series

### PR DESCRIPTION
**Pull request by phofl**
- [ ] closes #xxxx (Replace xxxx with the GitHub issue number)
- [x] [Tests added and passed](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#writing-tests) if fixing a bug or adding a new feature
- [x] All [code checks passed](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#pre-commit).
- [x] Added [type annotations](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#type-hints) to new arguments/methods/functions.
- [x] Added an entry in the latest `doc/source/whatsnew/vX.X.X.rst` file if fixing a bug or adding a new feature.

Follow-up on https://github.com/pandas-dev/pandas/pull/52947#discussion_r1187276744

### CODE REVIEW

File: pandas/tests/frame/test_constructors.py [hunk 1]
```diff
@@ -2657,6 +2657,12 @@ def test_construct_with_strings_and_none(self):
         expected = DataFrame({"a": ["1", "2", None]}, dtype="str")
         tm.assert_frame_equal(df, expected)
 
+    def test_construct_from_dict_ea_series(self):
```
Comment (jorisvandenbossche) [inline pandas/tests/frame/test_constructors.py:2660]: Small nit, but there is a `test_dict_nocopy` test a bit above that is specifically testing the behaviour of copy=True vs False for DataFrame(dict). Maybe can move the test to just below that one?

File: pandas/tests/frame/test_constructors.py [hunk 2]
```diff
@@ -2657,6 +2657,12 @@ def test_construct_with_strings_and_none(self):
...
+        # GH#53744
```
Comment (jorisvandenbossche) [inline pandas/tests/frame/test_constructors.py:2661]:
```suggestion
        # GH#53744 -  default of copy=True should also apply for Series with extension dtype
```

**Review by jorisvandenbossche — APPROVED** [review]

File: pandas/tests/frame/test_constructors.py (hunk 1 above)
Comment (phofl) [reply pandas/tests/frame/test_constructors.py]: Done

**Comment by jorisvandenbossche** [comment]
Thanks!

**Pull request merged**